tools/msvs/install_tools/install_tools.bat

```diff
@@ -3,21 +3,20 @@
 setlocal
 
 cls
+
 echo ====================================================
 echo Tools for Node.js Native Modules Installation Script
 echo ====================================================
 echo.
-echo This Boxstarter script will install Python and the Visual Studio Build Tools,
-echo necessary to compile Node.js native modules. Note that Boxstarter,
-echo Chocolatey and required Windows updates will also be installed.
+echo This script will install Python and the Visual Studio Build Tools, necessary
+echo to compile Node.js native modules. Note that Chocolatey and required Windows
+echo updates will also be installed.
 echo.
 echo This will require about 3 Gb of free disk space, plus any space necessary to
-echo install Windows updates.
-echo.
-echo This will take a while to run. Your computer may reboot during the
-echo installation, and will resume automatically.
+echo install Windows updates. This will take a while to run.
 echo.
-echo Please close all open programs for the duration of the installation.
+echo If the installation fails, please ensure Windows is fully updated and reboot
+echo your computer before trying again.
```

Comment:
Can we clarify exactly what this means?  Does this mean trigger the entire msi installation again?  Can they get back to the prompt to select the Chocolatey installation?  Or is this referring to manually triggering this .bat file?